portfolio/src/main/webapp/index.html

```diff
@@ -62,9 +59,23 @@ <h3>Overview</h3>
               <li>I hate raw tomatoes with a great passion</li>
               <li>I firmly believe there is a <em>Spongebob</em> quote for every situation</li>
             </ul>
+            <br>
+            <img src="images/me.jpg" alt="Picture of me">
+            <br>
           </div>
         </div>
       </div>
+      <footer class="page-footer font-xsmall">
+        <div class="container-fluid -fluid text-center text-md-left p-3 my-3 bg-dark text-white">
+          <div class="row">
+            <hr class="clearfix w-100 d-md-none pb-3">
+            <div class="col-md-3 mb-md-0 mb-3">
+              <h5 class="text-uppercase">Contact</h5>
+              <p>shandyk@google.com</p>
```

Comment:
optional: can we make this a <a> tag with "mailto:shandyk@google.com"?

Reply:
Done!